### ISSUE TITLE
Website changes: TrackablesTest Date errors 

### ISSUE THREAD

**Issue by bekuno** [cgeo/cgeo#15591]
New failing tests at CI:
~~~
12:29:26  cgeo.geocaching.connector.gc.TrackablesTest > testParseTrackableWithRetrievedDate[android-30(AVD) - 11] [31mFAILED [0m
12:29:26  	java.lang.AssertionError:
12:29:26  	Expecting actual not to be null
12:29:27  
12:29:27  cgeo.geocaching.connector.gc.TrackablesTest > testParseTrackableWithDiscoveredDate[android-30(AVD) - 11] [31mFAILED [0m
12:29:27  	java.lang.AssertionError:
12:29:27  	Expecting actual not to be null
~~~

**Comment by moving-bits** [comment]
This is a result of updating the TB sources (with a different user, that has not had the same interactions with this TB). 

I've cross-checked current TB source - it's somewhat different (`table` instead of `div` etc.), but the relevant line seems to be the same, thus I'll implant the old "discovered" / "retrieved" code block into the two relevant TB raw data files.

**Comment by moving-bits** [comment]
Fixed with #15594
